app/assets/stylesheets/forever_style_guide/modules/_nav-dropdowns.scss

```diff
@@ -68,6 +77,10 @@ $nav-icon-width: 60px;
     @include active-style($color-accent);
   }
 }
+//ugly cousin active state
```

Comment:
I like that idea in theory, but I'm having a hard time understanding how that would best work here since each potential emphasized navbar item would still need to be individually called out since they will have different active style colors.  Thoughts?